mrp_unbuild_subcontracting/models/mrp_unbuild.py

```diff
@@ -0,0 +1,79 @@
+import datetime
+
+from odoo import api, fields, models
+
+
+class MrpUnbuild(models.Model):
+    _inherit = "mrp.unbuild"
+
+    @api.model
+    def _get_default_date_planned_finished(self):
+        if self.env.context.get("default_date_planned_start"):
+            return fields.Datetime.to_datetime(
+                self.env.context.get("default_date_planned_start")
+            ) + datetime.timedelta(hours=1)
+        return datetime.datetime.now() + datetime.timedelta(hours=1)
+
+    @api.model
+    def _get_default_date_planned_start(self):
+        if self.env.context.get("default_date_deadline"):
+            return fields.Datetime.to_datetime(
+                self.env.context.get("default_date_deadline")
+            )
+        return datetime.datetime.now()
+
+    @api.model
+    def _get_default_picking_type(self):
+        company_id = self.env.context.get("default_company_id", self.env.company.id)
+        return (
+            self.env["stock.picking.type"]
+            .search(
+                [
+                    ("code", "=", "mrp_operation"),
+                    ("warehouse_id.company_id", "=", company_id),
+                ],
+                limit=1,
+            )
+            .id
+        )
```

Comment:
I think it would be great to use the same operation type used for subcontracting in standard odoo (warehouse_id.subcontracting_type_id), it will be more consistent.